tests/main.tf

```diff
@@ -0,0 +1,127 @@
+provider "aws" {
```

Comment:
The provider config ought to be unnecessary, as terraform will use defaults based on the ENVs...

Reply:
removed